skimage/measure/_regionprops.py

```diff
@@ -1109,10 +1120,14 @@ def regionprops(
 ):
     r"""Measure properties of labeled image regions.
 
+    Region properties are evaluated lazily. If you have a pre-determined list
+    of region properties (which you would like to print or feed),
+    consider using :func:`skimage.measure.regionprops_table` instead.
```

Comment:
I also don't know what "print or feed" means 😂 Suggestion:

> Region properties are evaluated on-demand and are of diverse types. If you want to do structured tabular data analysis of specific properties, consider using :func:`skimage.measure.regionprops_table` instead.

Reply:
I meant "print to screen or feed into a pandas DataFrame" (basically, consume in some way). I'd like to highlight the difference between `regionprops` (where you get the property values only when needed) and `regionprops_table` (where the results are right there, boom!).